Dockerfile-csi-node

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build stage
-FROM golang:1.12.6 as builder
+FROM golang:1.13 as builder
```

Comment:
Will it affect the entire node?
If so, I suggest bumping the version in a different PR.

Reply:
it will, but the reason to do it is coupled with this PR, and it's not a large diff, so I still think it is correct to include it here

Reply:
@ArbelNathan what do you think?

Reply:
I know we need it for this feature, but it can affect the entire node. Splitting it into two PRs can help us debug future issues.

Reply:
it will also make the relation between the changes less obvious and add a merge blocker overhead

Reply:
I agree with both but because our operator is already on 1.13 I'm not really worried and the connection here is important for context.